src/Wellcome.Dds/Wellcome.Dds.AssetDomainRepositories.Tests/Mets/MetsMetadataTests.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections.Generic;
+using FluentAssertions;
+using Wellcome.Dds.AssetDomainRepositories.Mets.Model;
+using Xunit;
+
+namespace Wellcome.Dds.AssetDomainRepositories.Tests.Mets
+{
+    public class MetsMetadataTests
+    {
+        [Fact]
```

Comment:
This would suit being a `[Theory]` rather than `[Fact]`, runs them as separate tests and get individual reporting on errors (2 of the 3 could pass and you'd see at a glance which ones)

```csharp
[Theory]
[InlineData("22mn 49s", 1369)]
[InlineData("1mn 41s", 101)]
public void TestDurationParsing(string input, int expected)
{
    PremisMetadata.ParseDuration(input).Should().Be(expected);
}
```

Reply:
Good point, I'd forgotten about that magic